cfme/test_framework/sprout/client.py

```diff
@@ -85,7 +85,9 @@ def __getattr__(self, attr):
     def from_config(cls, **kwargs):
         host = env.get("sprout", {}).get("hostname", "localhost")
         port = env.get("sprout", {}).get("port", 8000)
-        user_key = kwargs.pop('sprout_user_key') if 'sprout_user_key' in kwargs else None
+        user_key = kwargs.pop('sprout_user_key', None)
+        if user_key is None:
+            user_key = 'sprout'
```

Comment:
Will ask that we do _not_ do this, as it restores the default sprout user behavior that I removed just a while ago.  We want to force the user to explicitly pass a sprout user cred key, or have their env vars setup with their sprout creds